Assets/SEE/Net/Network.cs

```diff
@@ -273,7 +256,30 @@ private void Awake()
         }
 
         /// <summary>
-        /// Makes sure that we have only one <see cref="Instance"/>.
+        /// Name of command-line argument for room password <see cref="RoomPassword"/>.
+        /// </summary>
+        private const string passwordArgument = "-password";
+
+        /// <summary>
+        /// Name of command-line argument for UDP port <see cref="ServerPort"/>.
+        /// </summary>
+        private const string portArgument = "-port";
+        /// <summary>
+        /// Name of command-line argument for backend domain URL  <see cref="BackendDomain"/>.
+        /// </summary>
+        private const string domainArgument = "-domain";
```

Comment:
I'm being nitpicky here, but I think this should technically be `host` (see [RFC 1738](https://www.rfc-editor.org/rfc/rfc1738#section-3.1), for example) instead of `domain`.